packages/xhr-mock/src/MockXMLHttpRequest.ts

```diff
@@ -162,7 +162,7 @@ export default class MockXMLHttpRequest extends MockXMLHttpRequestEventTarget
 
     if (this.responseType === 'blob' && typeof body === 'string') {
       try {
-        throw notImplementedError;
+        return new Blob([body], {type: 'text/plain'});
```

Comment:
Is `Blob` provided by `whatwg-fetch`? Should we continue throwing the error when `Blob` isn't defined? Not sure if you can think of a clever way to test each case?

Reply:
whatwg-fetch doesn't provide Blob. If Blob isn't defined, an error will get thrown and fall back to the `return null` behavior of the `catch`. Returning null is the same behavior as when `throw notImplementedError` was there.